examples/timestamping/backend/src/lib.rs

```diff
@@ -39,35 +39,45 @@ pub use crate::{
 };
 
 use exonum::{
-    merkledb::BinaryValue,
+    merkledb::{validation::is_valid_index_name_component, BinaryValue},
     runtime::{CommonError, ExecutionContext, ExecutionError},
 };
 use exonum_derive::{ServiceDispatcher, ServiceFactory};
 use exonum_rust_runtime::{api::ServiceApiBuilder, Service};
+use exonum_supervisor::Configure;
 
 use crate::{api::PublicApi as TimestampingApi, schema::Schema};
 
 #[derive(Debug, ServiceDispatcher, ServiceFactory)]
-#[service_dispatcher(implements("TimestampingInterface"))]
+#[service_dispatcher(implements("TimestampingInterface", raw = "Configure<Params = Config>"))]
 #[service_factory(proto_sources = "proto")]
 pub struct TimestampingService;
 
+fn verify_config(context: &ExecutionContext<'_>, config: &Config) -> Result<(), ExecutionError> {
+    if !is_valid_index_name_component(&config.time_service_name) {
```

Comment:
Isn't this check redundant given that instance name sanitation is performed by the core? A service with an invalid name is guaranteed to never be instantiated.